docs/source/WritingTests.rst

```diff
@@ -227,10 +227,12 @@ you intend to create it.
          for that specific test and execution conditions (such as with or
          without variants).  Look for "Test data directories" in the test logs.
 
-.. note:: An older API, :attr:`avocado.core.test.Test.datadir`, allows access
+.. note:: An extint API, ``avocado.core.test.Test.datadir``, allowed access
```

Comment:
I guess you meant `extinct`, right?

Reply:
Perhaps I'd use `superseded` or `deprecated` to make it easier to understand...

Reply:
Not deprecated... as deprecated it was *before* this.  Let's use "A previously existing API, now removed, ... "